### PR TITLE
Improve event

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
See commit messages for what this PR improves:

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
screen shot 2018-09-27 at 14 57 04

File: source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb [hunk 1]
```diff
@@ -4,32 +4,39 @@ title:
...
   </div>
 </div>
 
 <section class="intro">
   <div class="row">
     <article>
+
       <p>
-        We nodigen je van harte uit om bij de Gebruikersbijeenkomst van Defacto aanwezig te zijn. We willen graag van deze gelegenheid gebruik maken om te laten zien wat <a href="/capp-lms/">de vernieuwde mogelijkheden</a> zijn met ons CAPP LMS.
+        We nodigen je van harte uit om bij de Gebruikersbijeenkomst van Defacto aanwezig te zijn. We willen graag van deze gelegenheid gebruik maken om te laten zien wat de vernieuwde mogelijkheden zijn met ons CAPP LMS.
```
Comment (fatboypunk) [inline source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb]: CAPP LMS also won't be linked here?

Comment (RonaldDijkstra) [reply source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb]: Can do!

File: source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb [hunk 2]
```diff
@@ -38,9 +45,16 @@ lang_ignore: en, de
           Voor de lunch wordt gezorgd
         </li>
       </ul>
+
+      <figure>
+        <%= image_tag "screenshots/capp-lms/course-catalog-browser_nl.png", title: "Cursuscatalogus", alt: "Cursuscatalogus" %>
```
Comment (fatboypunk) [inline source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb]: Might as well add a .webp 😶

Comment (RonaldDijkstra) [reply source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb]: Will do later, thanks.

File: source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb (hunk 1 above)
Comment (RonaldDijkstra) [reply source/localizable/evenementenagenda/gebruikersbijeenkomst-november-2018.html.erb]: Linked

**Pull request merged**